### ISSUE TITLE
Track : after switching to practice mode, i cannot switch back to learning mode

### ISSUE THREAD

**Issue by TomPradat** [exercism/exercism#6562]
I seem to be stuck on practice mode in the Javascript track (see the capture below)

![image](https://user-images.githubusercontent.com/16164512/193328942-6189a504-0b73-466a-a9a3-c5a93f63b59b.png)

I can only disable learning mode (see the menu), but this has no effect, which makes sense as I'm not in learning mode.

I also have no access to the syllabus tab, when clicking I get redirected to the current page (https://exercism.org/tracks/javascript). This works in other tracks where I'm in learning mode thought. Let me know if I should open another issue for this, I though this might be related.

Tell me if you need more information, I've tried switching browsers but have the problem on both Firefox (104.0.2) and Chrome (106.0.5249.61). I'm running on Ubuntu 22.04. I don't have errors in the web console.

Ping @ErikSchierboom (Jeremy asked me to ping you)

**Comment by ErikSchierboom** [comment]
@TomPradat I've forcibly disabled practice mode. Could you check to see if it is working?

**Comment by TomPradat** [comment]
> @TomPradat I've forcibly disabled practice mode. Could you check to see if it is working?

Yes I can see the "Syllabus" back again. Should I try to switch to "practice mode" again to see if I still have the issue ?

**Comment by ErikSchierboom** [comment]
@TomPradat Yes please!

**Comment by TomPradat** [comment]
@ErikSchierboom I just disabled learning mode and still have the issues :
1. I can't access the syllabus, I get redirected to `/tracks/javascript`
2. I cannot go back to practice mode, I only have "Disable Learning Mode" in the dropdown menu

**Comment by ErikSchierboom** [comment]
So weird. We'll look into it

**Comment by ErikSchierboom** [comment]
> I can't access the syllabus, I get redirected to /tracks/javascript

This is currently by design, as the syllabus is only shown when learning mode is disabled.

> I cannot go back to practice mode, I only have "Disable Learning Mode" in the dropdown menu

Which is the same as enabling Practice Mode :)

**Comment by ErikSchierboom** [comment]
@TomPradat We've found the issue! A PR has been opened to fix it: https://github.com/exercism/website/pull/3029
